.github/workflows/lint.yml

```diff
@@ -0,0 +1,44 @@
+name: Lint
```

Comment:
Prefere to move this in `sonar.yml`, I badly name this file but it correspond to our primary ci workflow :)